api-inference-community/docker_images/common/app/pipelines/text_to_image.py

```diff
@@ -0,0 +1,25 @@
+from app.pipelines import Pipeline
+
+
+class TextToImagePipeline(Pipeline):
+    def __init__(self, model_id: str):
```

Comment:
is the `model_id` a required argument? perhaps we can help the end-user by explaining what this is in the docstring?

Reply:
Yes, this is the docker image pipeline template (not the template shared to the user) that is copied when new libraries are added to the Hub.

I would prefer to keep this the same as the rest of the pipelines (see all other files under `pipelines/`) for the time being. We can send a different PR or commit to improve these. The end user of the `common` directory are library owners that are integrating to the Hub, so I don't think this is a big issue.

Reply:
ok sounds good to me, thanks for the explanation!